src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -68,8 +67,10 @@ class SgeBackend extends Backend with SharedFileSystem with LazyLogging {
       case SuccessfulExecution(_, _) => ExecutionStatus.Done
     }).toString
 
-  private def recordDatabaseFailure(call: Call, status: String, rc: Int): PartialFunction[Throwable, Unit] = {
-    case e: Throwable => logger.error(s"Failed to update database with ${call.name} status: $status, rc: $rc because $e")
+  private def recordDatabaseFailure(backendCall: BackendCall, status: String, rc: Int): PartialFunction[Throwable, Unit] = {
```

Comment:
This would probably be neater if it took the logger as an argument instead of DIYing it from the BackendCall